app/view_objects/teacher_interface/application_form_view_object.rb

```diff
@@ -253,9 +253,9 @@ def assessment_declined_reasons
                  assessor_feedback = failure_reason.assessor_feedback
                ).present? &&
                  FailureReasons.decline?(failure_reason: failure_reason.key)
-              "#{title}\n\n#{assessor_feedback}"
+              { decline_reason: title, assessor_note: assessor_feedback }
```

Comment:
Thinking about the structure of this actually, since it's coming from a method that's returning the "decline reasons" (which are made up of a name and optionally a note from the assessor) I reckon we should return the title under a `name` field:

```suggestion
              { name: title, assessor_note: assessor_feedback }
```